.gitignore

```diff
@@ -19,3 +19,4 @@
 /supervisord.pid
 /yarn-error.log
 /.php_cs
+/.DS_Store
```

Comment:
- No new line at end of file :'( (all scss files should have a new line too at the end no?)
- this line should be in your global gitignore configuration on your computer.

@Remg what are you thinking about that ?

Reply:
You should add this line in your `~/.gitignore_global` file. 
All files needed for your personal tools should not be added in the `.gitignore` project file.